source/org/zfin/fish/presentation/FishDetailController.java

```diff
@@ -77,22 +77,7 @@ protected String showFish(@PathVariable String zdbID, Model model) {
         model.addAttribute("phenotypeDisplays", PhenotypeService.getPhenotypeDisplays(phenotypeStatements, "condition", "phenotypeStatement"));
 
         // disease model

```

Comment:
I suppose we could get rid of these comments as well

Reply:
Yes.